src/sparql/illegal-annotation-property-violation.sparql

```diff
@@ -50,11 +50,11 @@ SELECT DISTINCT ?annotation WHERE {
     <http://purl.obolibrary.org/obo/UBPROP_0000201>,
     <http://purl.obolibrary.org/obo/UBPROP_0000202>,
     <http://purl.obolibrary.org/obo/term_editor>,
-    <http://purl.org/dc/elements/1.1/contributor>,
     <http://purl.org/dc/elements/1.1/creator>,
     <http://purl.org/dc/elements/1.1/date>,
     <http://purl.org/dc/elements/1.1/description>,
     <http://purl.org/dc/terms/source>,
+    <http://purl.org/dc/terms/contributor>,
```

Comment:
huh is there a reason to change to this? I mean just curious, i notice that it doesn't seem to affect the -edit file anyway

Reply:
This is needed to ensure this relation is even allowed I think!

Reply:
Yes, I changed from `dc_elements:contributor` to `dc_terms:contributor` to follow the recommendation. In the -edit file I also changed to dc_terms in the dc-contributor definition.

Reply:
Thanks!